lib/constellation.rb

```diff
@@ -0,0 +1,39 @@
+class ConstellationHttpException < StandardError
+end
```

Comment:
I see, so you mean that these other errors that might arise should be rescued/displayed as well? I guess if not, you'd get the standard Rails error page, which is pretty ugly.